Code/Framework/AzCore/AzCore/Math/Internal/SimdMathVec3_neon.inl

```diff
@@ -28,12 +28,12 @@ namespace AZ
 
         AZ_MATH_INLINE Vec3::FloatType Vec3::FromVec1(Vec1::FloatArgType value)
         {
-            return NeonQuad::FromVec1(value);
+            return NeonQuad::FromVec1(value); // {value.x, value.x, value.x, unused}
         }
 
         AZ_MATH_INLINE Vec3::FloatType Vec3::FromVec2(Vec2::FloatArgType value)
         {
-            return NeonQuad::FromVec2(value);
+            return NeonQuad::FromVec2(value); // {value.x, value.x, 0.0f, unused}
```

Comment:
Uh... should the comment be reference both **x and y**.
```suggestion
            return NeonQuad::FromVec2(value); // {value.x, value.y, 0.0f, unused}
```

Reply:
Yes. Fixed. Thanks.